documentation/asciidoc/topics/proc_creating_grafana_datasources.adoc

```diff
@@ -0,0 +1,65 @@
+[id='creating-grafana-dashboards_{context}']
+= Creating Grafana Data Sources
+Set up Prometheus as a data source so you can visualize {brandname} metrics in Grafana dashboards.
+
+.Prerequisites
+
+* Create a Prometheus service monitor for your {brandname} cluster.
+* Install the Grafana operator in your {brandname} cluster namespace.
+* Deploy a `Grafana` CR in your {brandname} cluster namespace.
```

Comment:
Again, this doesn't need to be in the cluster namespace. The user simply requires a Grafana deployment  to be created. This could be simplified to a single prerequisite that says something like "Install the Grafana operator and create a Grafana deployment".